### PR TITLE
Only display the install Juju button in sandbox mode

### LINKED ISSUE #1340: Remove "Install juju" button from standalone

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes #1340

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- load the gui locally
- check that there is no "Install Juju" button in the bottom right
- add a service
- the deploy button etc. should appear but the "Install Juju" button should still be hidden
- load the gui in a real env
- the "Install juju" button should now be visible

File: jujugui/static/gui/src/app/components/deployment-bar/test-deployment-bar.js [hunk 1]
```diff
@@ -45,13 +45,16 @@ describe('DeploymentBar', function() {
...
-    var services = sinon.stub();
+    var services = [];
     var renderer = jsTestUtils.shallowRender(
       <juju.components.DeploymentBar
         currentChangeSet={currentChangeSet}
         deployButtonAction={deployButtonAction}
         exportEnvironmentFile={sinon.stub()}
-        services={services} />, true);
+        importBundleFile={sinon.stub()}
+        renderDragOverNotification={sinon.stub()}
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/deployment-bar/test-deployment-bar.js]: A few props were added as a drive-by fix for missing propTypes warnings.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4440/
Test PASSed.

**Comment by hatched** [comment]
:+1: :ok: 

Thanks for this, definitely better.

**Comment by huwshimi** [comment]
Thanks @hatched :shipit:. As noted we'll have to bear in mind that this should not appear in the uncommitted model scenario.

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge
